templates/base.html.twig

```diff
@@ -3,7 +3,7 @@
 
 <head>
   <meta charset="utf-8">
-  <title>{% block title %}OCHA API{% endblock %}</title>
+  <title>{% block title %}OCHA Key Figures API{% endblock %}</title>
```

Comment:
Yes, this will change the text in the browser tab for all pages that don't override the title.